docs/developer-guide/ksqldb-reference/create-connector.md

```diff
@@ -24,6 +24,9 @@ configuration passed in the WITH clause. Some connectors have ksqlDB templates
 that simplify configuring them. For more information, see
 [Natively Supported Connectors](../../concepts/connectors.md#natively-supported-connectors).
 
+If the IF NOT EXISTS clause is present, the statement will fail if a connector with the supplied name

```

Comment:
Contrary, if the `IF NOT EXISTS` clause is present, the statement doesn't fail if the connector already exists.

Reply:
@spena I see , the current implementation in this PR throws an exception if a connector exists ,
so the expectation would be to simply pass along an `ErrorEntity` and skipping the connector creation part of the code ?

Reply:
Yes. 

Btw, there's another PR to support the DROP CONNECTOR IF EXISTS and included a `WarningEntity` class. It would be good to return that instead of the `ErrorEntity` (See https://github.com/confluentinc/ksql/pull/6067/files#diff-748287344db7330d94327cc33a1117e9)